### PR TITLE
Use respond module placeholder

### PR DESCRIPTION
**Pull request by ScriptedAlchemy**
implements #6

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -4,16 +4,23 @@ import {
...
-import {bindActionCreators} from 'redux'
+import { bindActionCreators } from 'redux';
+
 const useReduxActions = () => {
-  console.log('actionsGoHere')
-}
+  console.log('actionsGoHere');
+};
+
+const useRespond = module => ({
```
Comment (faceyspacey) [inline src/index.js]: good

File: src/babel/helpers.js [hunk 2]
```diff
@@ -81,17 +82,21 @@ export const isWrappedComponentSet = (
...
-      const state = _Remixx.useReduxState();
+export const makeWrappedComponent = (t, displayName) => t.expressionStatement(
+  t.assignmentExpression(
+    '=',
+    t.identifier(`const ${displayName}`),
+    t.identifier(`(props) => {
       const dispatch = _Remixx.useReduxDispatch();
-      const actions = _Remixx.useReduxActions();
-     
+       const { state, actions } = _Remixx.useRespond('__respond_pending_chunk_id__')
```
Comment (faceyspacey) [inline src/babel/helpers.js]: check

**Comment by faceyspacey** [comment]
solid. ..probably should move to `useRespond` soon, and forget the dispatch + binding. we can do that all under the hood.

**Comment by ScriptedAlchemy** [comment]
Ye will do, next on the list now

**Pull request merged**